spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/config/RabbitBinderConfigurationProperties.java

```diff
@@ -31,15 +31,15 @@
 
 	private int backOffMaxInterval;
 
-	private int backOffMultiplier;
+	private double backOffMultiplier;
 
 	private boolean transacted;
 
 	private boolean concurrency;
 
-	private MessageDeliveryMode defaultDeliveryMode;
+	private MessageDeliveryMode deliveryMode;
```

Comment:
The accessors are set/getDefaultDefaultDeliveryMode(). I checked this with Gary. This name was intentional for consistency